.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: scala
+
+# These directories are cached to S3 at the end of the build
+cache:
+  directories:
+    - $HOME/.ivy2/cache
+    - $HOME/.sbt/boot/
+
+jdk:
+  - oraclejdk8
+
+script:
```

Comment:
I can add it, I guess I could also ask for the Scala version when creating the template